packages/config/src/options.js

```diff
@@ -35,6 +35,12 @@ export function getNuxtConfig (_options) {
     options._routerBaseSpecified = true
   }
 
+  // TODO: Remove for Nuxt 3
+  // router.scrollBehavior -> app/router.scrollBehavior.js
+  if (options.router && typeof options.router !== 'undefined') {
+    consola.warn('`router.scrollBehavior` property is deprecated in favor of using `~/app/router.scrollBehavior.js` file')
```

Comment:
Can we probably guide users to the new docs for quick migration?

Reply:
Yes,  I will make a PR to the docs + link in the warn for it :)